### DISCUSSION TITLE
Developer mode

### DISCUSSION THREAD

**Discussion by gipi** [danielebruneo/remarkable2-hacks#17] [General]
**Be aware that disconnect the device from the account** probably use a different endpoint for the synchronization.

I discovered that is possible to enable a "developer mode" with the line ``ServiceEnvironment=Dev`` into ``/home/root/.config/remarkable/xochitl.conf`` (obviously you have to restart xochitl).

The following is what I see with the beta program, version 2.13.0.758, it seems that with the version after this stuff disappeared.

![developer](https://user-images.githubusercontent.com/1107413/171428760-513787d4-706e-4841-86fe-5f6c79ab79cd.png)
![experimental](https://user-images.githubusercontent.com/1107413/171428795-4edd03d6-06f0-42d5-aba4-ff42ac8c3e4b.png)

i don't know if is something already known or if you are interested in investigating more.

**Comment by danielebruneo** [comment]
Great finding @gipi !!!

**Comment by gipi** [comment]
I found that disconnect the device from the account, probably uses a different endpoint, if you remove the line and reconnect the account to the device there are no problems.

**Comment by DiBop** [comment]
I'm curious as to what Seabird is all about!

**Reply by gipi** [reply]
someone on Reddit indicated to have something to do with using a keyboard

**Comment by gipi** [comment]
It seems that with the new version (2.14) the experimental stuff disappeared.

Moreover, thanks to further investigations I discovered that are available other "environments" to indicate, like ``Staging``, ``Prod`` (that I think is the default), ``QA``. I think each one uses different endpoint for the APIs but haven't tried them.
